setup.py

```diff
@@ -326,16 +326,18 @@ def run(self):
     'onnxruntime.transformers.models.t5',
 ]
 
+package_data = {}
+data_files = []
+
+package_data["onnxruntime.tools.mobile_helpers"] = ["*.md", "*.config"]
```

Comment:
just curious, why move it up a few lines?

Reply:
to keep the initialization of `packages` close to `package_data` as the entry in `package_data` uses a value from `packages`.